cwms-data-api/src/main/java/cwms/cda/ApiServlet.java

```diff
@@ -285,6 +286,11 @@ public void init() {
                     logger.atInfo().withCause(e).log(re.toString(), e);
                     ctx.status(HttpServletResponse.SC_CONFLICT).json(re);
                 })
+                .exception(NoDataFoundException.class, (e, ctx) -> {
+                    CdaError re = new CdaError("Not Found.");
+                    logger.atInfo().withCause(e).log(re.toString());
+                    ctx.status(HttpServletResponse.SC_NOT_FOUND).json(re);
```

Comment:
There is already a NotFoundException - rather not bring org.jooq into this

Reply:
removed it